CMakeLists.txt

```diff
@@ -34,6 +34,12 @@ elseif(TARGET_CPU STREQUAL "aarch64")
     set(GCC_ARCH "armv8.2-a" CACHE STRING "Target architecture for aarch64")
 endif()
 
+if(GCC_ARCH STREQUAL "native")
+    set(M_TUNE "native" CACHE STRING "-mtune=native")
+else()
+    set(M_TUNE "generic" CACHE STRING "-mune=generic")
```

Comment:
Just `set(M_TUNE "generic" CACHE STRING "Tunes the generated code for the specified cpu-type")` without if-else is enough. People who want to set it to `native` or different should know better